actionpack/lib/action_dispatch/http/mime_negotiation.rb

```diff
@@ -139,6 +141,33 @@ def negotiate_mime(order)
         order.include?(Mime::ALL) ? format : nil
       end
 
+      class VariantInquirer # :nodoc:
+        delegate :==, :each, :empty?, to: :variants
+
+        def initialize(variants)
```

Comment:
`variants = []` to avoid `VariantInquirer.new([])` and use `VariantInquirer.new` instead